src/components/calcite-card/calcite-card.scss

```diff
@@ -1,4 +1,5 @@
 :host {
+  --calcite-border-radius: 0;
```

Comment:
Should this variable be named to reflect its just for calcite-card? Otherwise it seems like a global calcite-border-radius variable that would apply to other components.

Reply:
Should we document this variable in the readme? See the `calcite-alert` readme. https://github.com/Esri/calcite-components/blob/master/src/components/calcite-alert/readme.md#css-custom-propertiesw

Reply:
`--calcite-border-radius` is a preexisting global variable. It's used in modal etc

Reply:
Yeah I think it would be nice to document, not sure where though since its global. Did you document the icon color one somewhere?

Reply:
Interesting, I thought we documented all of them in the theme story. 

`6-custom-theme.stories.mdx`

All those global ones start with `--calcite-ui-`

Maybe we need to improve this documentation? A readme for global variables and consistent naming.

Reply:
It would also be beneficial to users if we let them know what variables are being used in a component. Global and non global.

Reply:
@jcfranco what do you think? ^

It seems like we need to document global variables and local variables better. It might also help to standardize the naming between them.

- global: `calcite-ui-`?
- local:  `calcite-<component>-`?

Reply:
On the developer site, we currently just show component-scoped variables: https://developers.arcgis.com/calcite-design-system/components/modal/#component-api-styles. I have a WIP feature that will let users override these in the component preview as well.

I do think it will be useful to also show "global css variables that can affect this component" so users can tell what effect global overrides will have. Right now it can be a bit of a guessing game as a consumer to determine which border or text color to override.

Reply:
I agree with both of you. If a user can use a global css var on a component, it should be documented on that component's page. We just haven't done that yet so I think we should be consistent and do them all at once. 

I will create an issue to audit which components use which global variables. And then we can add them side by side with the component-specific vars, or did you want to create a different doc section on the component pages for the global vars?

Reply:
> And then we can add them side by side with the component-specific vars, or did you want to create a different doc section on the component pages for the global vars?

It would be nice to assign them some distinguishing name / data so we can group them or otherwise indicate the difference in the docs.

Reply:
@benelan I totally edited your comment instead of mine - sorry 😆

Reply:
So I could refactor this one to the `-ui` pattern used by all of the other global ones. I don't think having them in a completely separate section on the doc page is necessary.

Reply:
This is probably something we should discuss in our meeting next week, I'll add it to the agenda

Reply:
> It would be nice to assign them some distinguishing name / data so we can group them or otherwise indicate the difference in the docs.

I think it would be super nice to easily know whether a variable is local to a component or global. I don't think we do anything like this right now.